tests/integration/service_test.py

```diff
@@ -699,6 +699,13 @@ def test_empty_labels(self):
         for name in labels_list:
             self.assertIn((name, ''), labels)
 
+    def test_custom_container_name(self):
```

Comment:
Is it worth us also having a test that demonstrates that an error is raised if you try to call scale with a named container?